<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ddml_ddlg</class>
 <widget class="QWidget" name="ddml_ddlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1570</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1570</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1570</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	
	color: rgb(255, 255, 255);

}

QDateTimeEdit{
	
	color: rgb(255, 255, 255);
}
QComboBox{
	color: rgb(255, 255, 255);
	font: 14pt &quot;黑体&quot;;
	border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
	
	background-color: rgba(3, 59, 96, 10);
}
QComboBox::down-arrow {

image:url(:/new/jsj/rc/新增/下拉箭头.png);

 }

 QComboBox::drop-down {

     border:none;

 padding-right:4px;

 }
QLineEdit{
border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */

	border-radius: 3px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */

font: 14pt &quot;黑体&quot;;
background-color: rgba(3, 59, 96, 80);
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1570</width>
     <height>290</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1570</width>
     <height>290</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1560</width>
     <height>290</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox{


	border-radius: 2px; /* 边框圆角 */
border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
	background-color: rgba(3, 59, 96);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1414</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>命令类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ml_type_Combo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
font: 14pt &quot;黑体&quot;;
	border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
}
QComboBox::down-arrow {

image:url(:/new/jsj/rc/新增/下拉箭头.png);

 }

 QComboBox::drop-down {

     border:none;

 padding-right:4px;

 }

QComboBox QAbstractItemView{

  outline:0px;

color:rgb(220,220,220);

background-color:rgb(37,86,159);

border:1px solid rgba(143,211,249,127);

border-bottom-right-radius:4px;

border-bottom-left-radius:4px;

 }

QComboBox QAbstractItemView::item{

height:26px;

background-color:rgba(143,211,249,38);

 }

 QComboBox QAbstractItemView::item:selected{

background-color:rgb(26,138,219);
 }
font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1300</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>301</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>发送时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="ml_Send_Time">
       <property name="minimumSize">
        <size>
         <width>185</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>185</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateTimeEdit{
font: 14pt &quot;黑体&quot;;
background:transparent;

border:1px solid #A0A0A0;

border-radius:2px;

min-height:30px;

max-height:30px;

}
</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>301</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>命令号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ml_NumberEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入5-6位命令号</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>251</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>调度员</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ml_Dd_Combo">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
font: 14pt &quot;黑体&quot;;
	border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
}
QComboBox::down-arrow {

image:url(:/new/jsj/rc/新增/下拉箭头.png);

 }

 QComboBox::drop-down {

     border:none;

 padding-right:4px;

 }

QComboBox QAbstractItemView{

  outline:0px;

color:rgb(220,220,220);

background-color:rgb(37,86,159);

border:1px solid rgba(143,211,249,127);

border-bottom-right-radius:4px;

border-bottom-left-radius:4px;

 }

QComboBox QAbstractItemView::item{

height:26px;

background-color:rgba(143,211,249,38);

 }

 QComboBox QAbstractItemView::item:selected{

background-color:rgb(26,138,219);
 }</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>179</y>
      <width>271</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>调度台</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ml_ddt_Combo">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>300</y>
     <width>1580</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1580</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1552</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 2px; /* 边框圆角 */
border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
font: 18pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
	background-color: rgba(3, 59, 96);
</string>
   </property>
   <property name="placeholderText">
    <string>请手动输入调度内容</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
